File: examples-sdk/csharp/024-basic-llm-chat/Main.cs
// 024-basic-llm-chat — C# SDK equivalent
// Compile: vil compile --from csharp --input 024-basic-llm-chat/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("llm-chat", 8080);
var chat = new ServiceProcess("chat");
chat.Endpoint("POST", "/chat", "chat_handler");
server.Service(chat);
server.Compile();

File: examples-sdk/swift/024-basic-llm-chat/main.swift
// 024-basic-llm-chat — Swift SDK equivalent
// Compile: vil compile --from swift --input 024-basic-llm-chat/main.swift --release

let server = VilServer(name: "llm-chat", port: 8080)
let chat = ServiceProcess(name: "chat")
chat.endpoint(method: "POST", path: "/chat", handler: "chat_handler")
server.service(chat)
server.compile()
